src/Connection.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Tighten\SolanaPhpSdk;
+
+use Tighten\SolanaPhpSdk\Exceptions\AccountNotFoundException;
+use Tighten\SolanaPhpSdk\Util\Commitment;
+use Tighten\SolanaPhpSdk\Util\Ed25519Keypair;
+
+class Connection extends Program
+{
+    /**
```

Comment:
Same note RE doc blocks for all of these; if we're type hinting I think we can drop the doc blocks unless they provide any value you know of?

Reply:
Visual separation 🤩